aten/src/ATen/native/native_functions.yaml

```diff
@@ -5332,6 +5332,31 @@
     NestedTensorCPU, NestedTensorCUDA: NestedTensor_from_padded_and_nested_example
   autogen: _nested_from_padded_and_nested_example.out
 
+- func: _to_buffer(Tensor(a) self) -> Tensor(a)
+  variants: method
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_to_buffer
+
+- func: _to_buffer_copy(Tensor self) -> Tensor
+  variants: method
+  tags: view_copy
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: NestedTensor_to_buffer_copy
+
+- func: _from_buffer_maybe_view(Tensor(a) self, Tensor reference) -> Tensor(a)
```

Comment:
If this version is the one that most people should use, we should give it the most obvious name.